numba/core/config.py

```diff
@@ -385,12 +385,6 @@ def avx_default():
         CUDA_PER_THREAD_DEFAULT_STREAM = _readenv(
             "NUMBA_CUDA_PER_THREAD_DEFAULT_STREAM", int, 0)
 
-        # Compute contiguity of device arrays using the relaxed strides
-        # checking algorithm.
-        NPY_RELAXED_STRIDES_CHECKING = _readenv(
-            "NUMBA_NPY_RELAXED_STRIDES_CHECKING",
-            int, 1)
-
         # HSA Configs
```

Comment:
Can probably remove this HSA config var as ROCm has been removed entirely now the deprecation notice has gone!

Reply:
Many thanks.